### DISCUSSION TITLE
Does anyone know if streaming through the API counts towards Twitch-tracked watchtime stats that they started doing?

### DISCUSSION THREAD

**Discussion by danglingptr0x0** [streamlink/streamlink-twitch-gui#996] [Q&A] [answered]
May not be important to everyone, but I consider it quite essential to have my watchtime tracked so that I could then see accurate stats when Twitch generates them (annually).

**Comment by Coldblackice** [comment]
Also wondering the same. I know it definitely does not if an external proxy is utilized, but unsure of the specifics of the tracking mechanism, i.e. is the tracking browser/JS-based.

**Reply by fuguesoft** [reply, marked as answer]
I have other machines with tabs open if I want my metrics as watching through mpv even with the Twitch chat popout going does not dispense channel points or update the year in review

Streamlink also does not follow raids
